packages/hint-axe/scripts/create/utils.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs');
+const path = require('path');
+const { startCase } = require('lodash');
+
+/**
+ * @typedef {{
+ *     id: string;
+ *     metadata: {
+ *         description: string;
+ *         help: string;
+ *     },
+ *     tags: string[]
+ * }} RuleMeta
+ */
+
+/**
+ * @param {string} str
+ */
+const capitalize = (str) => {
+    if (str === 'aria') {
+        return 'ARIA';
+    }
+
+    return startCase(str).replace(/\band\b/i, 'and');
+};
+
+/**
+ * @param {string} category
+ */
+const categoryId = (category) => {
+    return category.replace('cat.', '');
+};
+
+/**
+ * @param {string} id
+ */
+const escapeKey = (id) => {
+    return id.includes('-') ? `'${id}'` : id;
+};
+
+/**
+ * @param {RuleMeta} rule
+ */
+const isRuleDisabled = (rule) => {
+    return rule.tags.includes('experimental') || rule.tags.every((tag) => {
+        return !tag.startsWith('wcag');
+    });
+};
+
+/**
+ * @param {RuleMeta} rule
+ */
+const isRuleEnabled = (rule) => {
+    return !isRuleDisabled(rule);
+};
+
+/**
+ * @param {string} packagePath
+ */
+const readFile = (packagePath) => {
+    const filename = path.resolve(`${__dirname}/../../${packagePath}`);
+
+    return new Promise((resolve, reject) => {
```

Comment:
any reason to not use `util.promisify`?

Reply:
`util.promisify` drops type information IIRC.